src/components/List/List.tsx

```diff
@@ -1,90 +1,157 @@
-import React, { memo } from 'react';
-import isEqual from 'react-fast-compare';
+import { useFocusRing } from '@react-aria/focus';
+import { useListBox, useOption } from '@react-aria/listbox';
+import { mergeProps } from '@react-aria/utils';
+import { useListState } from '@react-stately/list';
+import React from 'react';
+import { AriaListBoxProps, useListBoxSection } from 'react-aria';
 import { TestProps } from 'utils/types';
 
-import { wrapperStyle } from './List.style';
-import NormalList from './NormalList';
-import { ListItemType, ListRowSize, SelectHandlerType } from './types';
-import VirtualizedList from './VirtualizedList';
+import ListItemWrapper from './components/ListItemWrapper/ListItemWrapper';
+import { ListItemWrapperStyled } from './components/ListItemWrapper/ListItemWrapper.style';
+import { listStyle, wrapperStyle } from './List.style';
+import { ListSelected, ListSelection } from './types';
+import Window from './Window';
+import useCombinedRefs from '../../hooks/useCombinedRefs';
+import { SelectOption } from '../Select';
 
 export type ListProps = {
-  /** Data for the list */
-  data: ListItemType[];
-  /** Size of the list's row (height of ListItem Component)  */
-  rowSize: ListRowSize;
+  label: string;
```

Comment:
(trivial but) can you add a comment description for this prop?

Reply:
solved at [`3b16bc4` (#655)](https://github.com/Orfium/orfium-ictinus/pull/655/commits/e66fc4cf6560ef0e71db655c084ebbf3d27ed680)